src/test/java/org/takes/facets/fork/FkContentTypeTest.java

```diff
@@ -59,14 +62,17 @@
     @Test
     public void matchesWithAnyTypes() throws IOException {
         MatcherAssert.assertThat(
-            new FkContentType("text/xml", new RsEmpty()).route(
-                new RqWithHeader(
-                    new RqFake(),
-                    FkContentTypeTest.CONTENT_TYPE,
-                    "*/* "
-                )
-            ).has(),
-            Matchers.is(true)
+            new FkContentType(
+                "text/xml",
+                FkContentTypeTest.createTakeWithEmptyResponse()
+                ).route(
```

Comment:
@dalifreire there's something wrong with indentation here, this line should be 4 spaces back